cirq-google/cirq_google/api/v2/results.py

```diff
@@ -179,14 +185,16 @@ def results_from_proto(
 
     measure_map = {m.key: m for m in measurements} if measurements else None
     return [
-        _trial_sweep_from_proto(sweep_result, measure_map) for sweep_result in msg.sweep_results
+        _trial_sweep_from_proto(sweep_result, job, measure_map)
+        for sweep_result in msg.sweep_results
     ]
 
 
 def _trial_sweep_from_proto(
     msg: result_pb2.SweepResult,
+    job: 'cg.engine.AbstractJob',
```

Comment:
Add Optional here.  
Normally, would complain about adding a paremeter here would break backwards compatibility, but this is a private function so it should be fine.  =

Reply:
no longer optional